sound/soc/intel/boards/sof_sdw.c

```diff
@@ -588,14 +588,15 @@ static void init_dai_link(struct device *dev, struct snd_soc_dai_link *dai_links
 			  struct snd_soc_dai_link_component *cpus, int cpus_num,
 			  struct snd_soc_dai_link_component *codecs, int codecs_num,
 			  int (*init)(struct snd_soc_pcm_runtime *rtd),
-			  const struct snd_soc_ops *ops)
+			  const struct snd_soc_ops *ops, bool nonatomic)
```

Comment:
one more args for init_dai_link. Can we make it "static inline void", then args count is not a problem.